command-expansion-server/src/app.ts

```diff
@@ -17,15 +31,35 @@ app.use(bodyParser.json({ limit: '25mb' }));
 
 DbExpansion.init();
 const db = DbExpansion.getDb();
-const graphqlClient = new GraphQLClient(getEnv().MERLIN_GRAPHQL_URL);
 
-const logger = getLogger("app");
+const piscina = new Piscina({ filename: `${path.resolve()}/build/worker.js` });
+
+type Context = {
+  commandTypescriptDataLoader: InferredDataloader<typeof commandDictionaryTypescriptBatchLoader>,
+  activitySchemaDataLoader: InferredDataloader<typeof activitySchemaBatchLoader>,
+  simulatedActivityInstanceDataLoader: InferredDataloader<typeof simulatedActivityInstanceBatchLoader>,
+  expansionSetDataLoader: InferredDataloader<typeof expansionSetBatchLoader>,
+};
+
+app.use(async(req: Request, res: Response, _next: NextFunction) => {
+  const graphqlClient = new GraphQLClient(getEnv().MERLIN_GRAPHQL_URL);
+
+  const context: Context = {
+    commandTypescriptDataLoader: new DataLoader(commandDictionaryTypescriptBatchLoader({graphqlClient})),
```

Comment:
I forgot about this, we need to add a cache-key function to each of these.

The line 48 should be 
```
    commandTypescriptDataLoader: new DataLoader(commandDictionaryTypescriptBatchLoader({graphqlClient}), {cacheKeyFn: objectCacheKeyFunction}),

```
and the rest (lines 49-51) should similarly add that second parameter to set the cacheKeyFn.

WHAT THIS DOES:
Adjusts how the cache-key is created and checked in the dataloader for in-request caching. The object cache-key function just concatenates the object parameters together as the cache key.